src/Microsoft.Health.Dicom.Tests.Common/Comparers/CustomTagEntryEqualityComparer.cs

```diff
@@ -0,0 +1,34 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using Microsoft.Health.Dicom.Core.Features.CustomTag;
+
+namespace Microsoft.Health.Dicom.Tests.Common.Comparers
+{
+    public class CustomTagEntryEqualityComparer : IEqualityComparer<CustomTagEntry>
```

Comment:
A typical pattern for comparers is to provide a static instance for people to use instead of each class keeping their own cached instance. Eg.

```csharp
// Could also be called "Instance?"
public static readonly CustomTagEntryEqualityComparer Default { get; } = new CustomTagEntryEqualityComparer();
```

Reply:
will fix it.

Reply:
`public` fields are not recommended, even `static` ones. One of the reasons is that it's harder to make changes to the API going forward, as properties are glorified methods whose implementation can change.

For example, `Comparer<T>.Default` uses this pattern: https://docs.microsoft.com/en-us/dotnet/api/system.collections.generic.comparer-1.default?view=net-5.0#System_Collections_Generic_Comparer_1_Default